WordPress/WordPressStatsWidgets/LockScreenWidgets/Models/LockScreenWidgetViewModelMapper.swift

```diff
@@ -3,12 +3,17 @@ import Foundation
 struct LockScreenWidgetViewModelMapper {
     let data: HomeWidgetData
 
-    func getLockScreenSingleStatViewModel(title: String, dateRange: String) -> LockScreenSingleStatViewModel {
+    func getLockScreenSingleStatViewModel(
+        title: String,
+        dateRange: String,
+        footer: String
+    ) -> LockScreenSingleStatViewModel {
         LockScreenSingleStatViewModel(
             siteName: getSiteName(),
             title: title,
             value: getViews(),
             dateRange: dateRange,
+            footer: footer,
```

Comment:
What do you think about naming it by purpose and not by location? I would say this is actually the `title`.

Reply:
Oh yeah, you're right, originally I was thinking the name to separate from title and dateRange, so I choose footer 😅
but actually, as you said, we just need title for now.